overlord/configstate/applyconfighandler.go

```diff
@@ -19,7 +19,15 @@
 
 package configstate
 
-import "github.com/snapcore/snapd/overlord/hookstate"
+import (
+	"fmt"
+
+	"github.com/snapcore/snapd/overlord/hookstate"
+)
+
+// CachedTransaction is the index into the context cache where the initialized
+// transaction is stored.
+type CachedTransaction struct{}
```

Comment:
Yeah good call, okay, in the latest push I essentially moved `getTransaction` to `ContextTransaction`.